lib/MusicBrainz/Server/Filters.pm

```diff
@@ -224,6 +224,7 @@ sub format_editnote
          [<a href="//$server/edit/$1">edit #$1</a>]gi;
 
     # links to wikidocs
+    # (only safe because \w doesn't match any of the HTML reserved characters)
```

Comment:
I thought this should be documented, in case someone at some future point intends to change the regex.